commands_common/flags.go

```diff
@@ -33,7 +33,7 @@ func InputFileFlag() cli.Flag {
 		Name:     "input",
 		Aliases:  []string{"i"},
 		Usage:    "[Mandatory] Input file",
-		Required: true,
+		Required: false,
```

Comment:
in this case , you making it inconsistent with the text, is it mandatory or not:)? and please pass this as input to the function